src/webhint-theme/source/core/css/layouts.css

```diff
@@ -115,39 +115,39 @@ ol.layout {
 
 @media (min-width: 48em) {
 
-    [class*="layout--sidebar"] > .module {
+    [class*="layout--sidebar"] > .module--wrapper > .module {
```

Comment:
We can't remove `[class*="layout--sidebar"] > .module` on any of these because this affects other pages throughout the site, we should just add the code you added after the original code, like this: 

```
[class*="layout--sidebar"] > .module,
[class*="layout--sidebar"] > .module--wrapper > .module {

}
```